docs/rpc.md

```diff
@@ -45,9 +45,9 @@ curl -s -X POST http://127.0.0.1:5000 -H 'Content-Type: application/json' -d \
 Discover peers on the network via DHT that have active swap offers.
 
 Parameters:
-- `provides` (optional): one of `ETH` or `XMR`, depending on which offer you are searching
+- `provides`: (optional) one of `ETH` or `XMR`, depending on which offer you are searching
```

Comment:
I was just harmonizing the style. I don't think it matters which side of the colon the `(optional)` is on, as long as they all do it the same way.